Flick/Group/GroupResultViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  GroupResultViewController.swift
+//  Flick
+//
+//  Created by Haiying W on 1/25/21.
+//  Copyright © 2021 flick. All rights reserved.
+//
+
+import UIKit
+
+class GroupResultViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let resultsTableView = UITableView()
+    private let votingStatusImageView = UIImageView()
+    private let votingStatusLabel = UILabel()
+
+    // MARK: - Private Data Vars
+    private let votingResultCellReuseIdentifier = "votingResultCellReuseIdentifier"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .offWhite
+
+        votingStatusImageView.image = UIImage(named: "stillVotingIcon")
```

Comment:
idk if this is better but I believe you can just do like  `private let votingStatusImageView = UIImageView(image: UIImage(named: "stillVotingIcon"))` I like this cause its more succinct but also like a lot of things all in one line